app/mailers/observer_mailer.rb

```diff
@@ -1,28 +1,43 @@
 class ObserverMailer < ApplicationMailer
-  def on_observer_added(observation, reason)
+  def observer_added_confirmation(observation, reason)
```

Comment:
ah ok. I was trying to follow your convention but did not understand the correlation between `to` and the name. Agreed this one should change.

I think its fine to have that convention; I have tried to avoid `email` in all mailer method names for the redundancy reasons.